metis-repository/src/main/java/eu/europeana/metis/repository/rest/HttpHarvestController.java

```diff
@@ -0,0 +1,86 @@
+package eu.europeana.metis.repository.rest;
+
+import eu.europeana.metis.repository.dao.Record;
+import eu.europeana.metis.repository.dao.RecordDao;
+import eu.europeana.metis.utils.RestEndpoints;
+import io.swagger.annotations.Api;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import io.swagger.v3.oas.annotations.tags.Tags;
+import java.io.BufferedOutputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+import java.util.List;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.commons.compress.utils.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Controller for HTTP harvesting.
+ */
+@RestController
+@RequestMapping("http")
```

Comment:
Also this line should be removed because you already define the full mapping below (in the individual request(s)).

Reply:
Removed